app/models/manageiq/providers/container_manager.rb

```diff
@@ -30,5 +30,10 @@ def aggregate_cpu_total_cores(targets = nil)
     def aggregate_memory(targets = nil)
       aggregate_hardware(:computer_systems, :memory_mb, targets)
     end
+
+    # enables overide of ChartsLayoutService#find_chart_path
+    def chart_layout_path
+      "ManageIQ_Providers_ContainerManager"
+    end
```

Comment:
I like this better... maybe we'll find a better place to put this, such as a decorator or something similar... but I like this better than the first approach.  What do you think @simon3z?